src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --color-link: #00acdc;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  margin: 10px 50px;
+}
+
+.nav__li {
+  list-style-type: none;
+  display: inline;
+}
+
+.nav__link {
+  text-decoration: none;
+  margin-left: 17px;
+  color: #000;
+  display: inline-block;
+  position: relative;
+  padding: 22.5px 0;
+}
+
+.nav__links {
+  margin: 0 50px 0 23px;
+  display: inline-block;
+
+}
+
+.nav__link:hover {
+  color: var(--color-link);
+
+}
+
+.is-active {
+  color: var(--color-link);
+}
+
+.is-active::after {
+  margin: 0;
+  color: var(--color-link);
+  position: absolute;
+  /* stylelint-disable-next-line string-quotes */
+  content: '';
+  height: 4px;
+  bottom: -3px;
+  border-radius: 8px;
+  left: 0;
+  right: 0;
+  width: 100%;
+  background: var(--color-link);
+}
+
+.nav__link:focus::after {
```

Comment:
The color property is not needed in .nav__link:focus::after pseudoclass and should be removed